src/sentry/integrations/opsgenie/client.py

```diff
@@ -92,7 +92,17 @@ def send_notification(self, data, rules=None):
             event = data
             payload = self._get_issue_alert_payload(data, rules, event, group)
         else:
-            # this is for metric alerts, which will be in the next PR
-            pass
-        headers = {"Authorization": "GenieKey " + self.integration_key}
-        return self.post("/alerts", data=payload, headers=headers)
+            # if we're acknowledging the alert
+            if data.get("identifier"):
+                alias = data["identifier"]
+                resp = self.post(
+                    f"/alerts/{alias}/acknowledge",
+                    data={},
+                    params={"identifierType": "alias"},
+                    headers=headers,
+                )
+                return resp
```

Comment:
Can you expand on what this means with context to sentry? What are we acknowledging here and what might trigger this code path? Also where does the `identifier` key come from, in case someone was trying to trace things back?

Reply:
Left a comment explaining this, but "acknowledge" is the equivalent of resolving an alert in PagerDuty; i.e. the alert can be reopened. The identifier is the `alert_key` that we create to identify the alert.